jenkins/gradle-check-assemble.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+IFS=`echo -ne "\n\b"`
+
+docker ps
+docker stop `docker ps -qa` > /dev/null 2>&1
+
+cd search
+
+echo "./gradlew check --no-daemon --no-scan"
+./gradlew check --no-daemon --no-scan
+GRADLE_CHECK_STATUS=$?
+
+
+# Archive reports
+cd ../
+REPORTS_DIR=`find ./search -name reports`
+zip -9 -q -r gradle_check_${BUILD_NUMBER}_reports.zip ${REPORTS_DIR}
+ls | grep "gradle_check_${BUILD_NUMBER}"
```

Comment:
Not a blocker for this PR, just a thought:
Looks like we can publish code coverage results from here. 
cc: @tlfeng